test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -2615,14 +2670,17 @@ static void traffic_mngr_test_wred_profile(void)
 
 static int set_shaper(const char    *node_name,
 		      const char    *shaper_name,
-		      const uint64_t commit_bps,
-		      const uint64_t commit_burst_in_bits)
+		      uint64_t      commit_bps,
+		      uint64_t      commit_burst_in_bits)
 {
 	odp_tm_shaper_params_t shaper_params;
 	odp_tm_shaper_t        shaper_profile;
 	odp_tm_node_t          tm_node;
 	int rc;
 
+	commit_bps = clamp_rate(commit_bps);
+	commit_burst_in_bits = clamp_burst(commit_burst_in_bits);
```

Comment:
It seems that some of the call sites of set_shaper() may not be prepared to have the rate and burst parameters getting clamped. I am looking at these:

test_sched_queue_priority()
test_sched_node_priority()
test_sched_wfq()
test_byte_wred()
test_pkt_wred()
test_query_functions()

They all use specific burst sizes and I wonder if it is ok to just change it somewhat arbitrarily without possibly breaking the tests. Also, the rate gets clamped, but the functions later use the unclamped rate when receiving packets. receive_pkts calculates RX loop timeout based on the rate. I think there might be too early timeouts if the rate gets clamped downward too much since then the RX loop does not know to be patient enough.